### DISCUSSION TITLE
Version 0.2.0

### DISCUSSION THREAD

**Discussion by siku2** [siku2/hass-vzug#6] [Announcements]
## What's Changed

- Expose all 'categories' and 'commands' (i.e. user settings) as entities.
- Use 'getEcoInfo' command instead of puzzling together the eco info from the commands.
- Config entries no longer break if given just an IP address.

**Full Changelog**: https://github.com/siku2/hass-vzug/compare/0.1.2...0.2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/siku2/hass-vzug/releases/tag/0.2.0'>Version 0.2.0</a>.</em>

**Comment by IncendioMoussy** [comment]
Work to my ZUG AdoraLaver V4000

good Job

**Reply by IncendioMoussy** [reply]
I like you integer French translation:-). THX 



fr.json
```json
{
    "config": {
        "abort": {
            "already_configured": "Le périphérique est déjà configuré"
        },
        "error": {
            "cannot_connect": "Échec de la connexion",
            "unknown": "Erreur inconnue"
        },
        "step": {
            "user": {
                "data": {
                    "host": "Nom d'hôte (ou IP)"
                }
            }
        }
    },
    "entity": {
        "button": {
            "check_update": {
                "name": "Vérifier les mises à jour"
            }
        },
        "sensor": {
            "energy_program": {
                "name": "Consommation électrique "
            },
            "energy_average": {
                "name": "Consommation moyenne électrique"
            },
            "energy_total": {
                "name": "Consommation total électrique"
            },
            "water_program": {
                "name": "Consommation eau"
            },
            "water_average": {
                "name": "Consommation moyenne eau"
            },
            "water_total": {
                "name": "Consommation total eau"
            },
            "last_notification": {
                "name": "Dernière notification"
            },
            "program": {
                "name": "Programme",
                "state": {
                    "standby": "Veille"
                }
            },
            "program_end": {
                "name": "Fin du programme"
            },
            "program_end_raw": {
                "name": "Valeur estimatif de la fin lavage"
            },
            "status": {
                "name": "état"
            }
        },
        "update": {
            "update": {
                "name": "Mise à jour"
            }
        }
    }
}
```

**Reply by siku2** [reply]
Thanks, @IncendioMoussy. I went ahead and added it to the Repo in ebe931b9a0b5f76a795ad6ba9954e82b479eea28. Don't hesitate to open a pull request for these things though, as it makes it easier to integrate into the repo while still giving you credit for the changes :)

Will release a new version with this in a few moments.

**Reply by IncendioMoussy** [reply]
Thx
I am install 0.3.0, fils translation fr is installe, but is no used , the text is in english , i send capture if you like

**Reply by siku2** [reply]
@IncendioMoussy I assume you're talking about the entity names? Entity names are only translated if you set your Home Assistant's overall language to French (NOT just your user preference).

If you only change it for your user, it will only affect anything that isn't the entity name like the config entry flow:
![image](https://github.com/siku2/hass-vzug/assets/16530283/e1b1e39d-807c-4c42-9cbe-395960c8ca5a)

or the "standby" state of the program sensor:
![image](https://github.com/siku2/hass-vzug/assets/16530283/5b546d08-2e75-4b27-8eaa-d7e73f0f1db3)

If you change the general Home Assistant language though, it will start looking like this:
![image](https://github.com/siku2/hass-vzug/assets/16530283/8810ccdb-fff1-4350-8510-d745397b51b6)


The reason being that the name is unfortunately still an integral part of some of Home Assistant's internal systems, so it's not possible to change it in the integration.

**Reply by siku2** [reply]
Btw, you can find the global language under "Système > Général" :
![image](https://github.com/siku2/hass-vzug/assets/16530283/00d5d47b-83f4-47d8-8331-5783dfca6cdf)
